### PR TITLE
Tooltips and Aria labels

### LINKED ISSUE #2564: Icons should have hover tooltips & aria labels for accessibility
**What existing feature could be improved?**
The share, projection, camera, info and measurement tools do not describe what they are. Having a tooltip describe the button on hover would be helpful to new users and those that are impaired. Additionally, we should add aria-labels to these buttons for enhanced accessibility.

**Describe the improvement you'd like**
- Tooltips on icon hover
- Aria labels on icons/buttons

**Describe alternatives you've considered**
None.

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by jasontk19**
## Description

Fixes #2564 

- [x] Convert font awesome icon use to single-import library style 
    * Check out `web/js/font-awesome-library.js` for details
- [x] Add tooltips and aria labels to most interactive elements that are represented by an icon without text

### CODE REVIEW

File: web/js/components/layer/product-picker/search/filter-chips.js [hunk 1]
```diff
@@ -31,7 +30,7 @@ export default function FilterChips(props) {
         >
           {displayValue}
           <FontAwesomeIcon
-            icon={faTimes}
+            icon="timies"
```
Comment (edplato) [inline web/js/components/layer/product-picker/search/filter-chips.js:33]:
```suggestion
            icon="times"
```

Comment (Benjaki2) [inline web/js/components/layer/product-picker/search/filter-chips.js:33]:
```suggestion
            icon="times"
```

File: web/js/font-awesome-library.js [hunk 2]
```diff
@@ -0,0 +1,134 @@
+/*
+   Provides a single import location for font-awesome icons.  This allows us
+   to simply import the FontAwesomeIcon component where we want to display
+   an icon and only pass the string name of an icon.
+
+   For example:
+      import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+      <FontAwesomeIcon icon="times" fixedWidth />
```
Comment (Benjaki2) [inline web/js/font-awesome-library.js:8]: Just out of curiosity... Is this a more performant solution, or more of an enhancement in terms of future development?

File: web/js/containers/animation-widget.js [hunk 3]
```diff
@@ -441,14 +359,81 @@ class AnimationWidget extends React.Component {
...
+    const elemExists = document.querySelector('#create-gif-button');
+    const showWarning = elemExists && numberOfFrames >= maxFrames;
+    const warningMessage = (
+      <span>
+        Too many frames were selected.
+        <br />
+        Please request less than 40 frames if you would like to generate a GIF.
+      </span>
+    );
+    const labelText = 'Create an animated gif';
```
Comment (edplato) [inline web/js/containers/animation-widget.js:392]: 😄 
```suggestion
    const labelText = 'Create an animated GIF';
```

**Review by Benjaki2 — COMMENTED** [review]
Looks good! 👍  one minor spelling change requested.

**Review by edplato — CHANGES_REQUESTED** [review]
Nice, I like the tooltips and the FontAwesome import consolidation. I encountered an issue with the alert component activated from hiding the tour - it is missing its exclamation icon and a console error gets thrown. I think this can be fixed by changing the `iconClassName` props in `renderSupportAlert` and `renderDisableAlert` in `web\js\containers\tour.js` to `""` from `" "`.

File: web/js/components/layer/product-picker/search/filter-chips.js (hunk 1 above)
Comment (jasontk19) [reply web/js/components/layer/product-picker/search/filter-chips.js:33]: Nice, good catch!

File: web/js/font-awesome-library.js (hunk 2 above)
Comment (jasontk19) [reply web/js/font-awesome-library.js:8]: I don't think there is a difference in performance but I think only having to ever import an icon once seemed tidier.  That way, for any component that needs to show an icon, you only need to ever import `<FontAwesomeIcon>`

**Review by Benjaki2 — APPROVED** [review]
:

**Comment by jasontk19** [comment]
@edplato I missed the issue in your comment on my first pass;  should be good now!

**Review by edplato — APPROVED** [review]

**Comment by edplato** [comment]
Looks good! 🚀

**Pull request merged**